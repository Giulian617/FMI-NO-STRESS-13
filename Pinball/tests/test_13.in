52 86887
17 34 3
10 17 7
15 10 7
38 15 8
14 38 1
12 14 8
52 12 7
27 52 10
36 27 8
25 36 6
50 25 4
1 50 1
23 1 6
28 23 4
37 28 7
35 37 4
31 35 2
18 31 5
21 18 2
13 21 9
6 13 7
45 6 5
7 45 9
30 7 7
19 30 5
8 19 2
5 8 5
2 5 4
16 2 9
24 16 1
48 24 2
46 48 2
22 46 1
42 22 8
4 42 2
20 4 9
40 20 10
51 40 4
47 51 2
26 47 5
44 26 5
43 44 2
32 43 4
29 32 6
9 29 10
3 9 9
11 3 2
33 11 1
49 33 8
41 49 3
39 41 1
