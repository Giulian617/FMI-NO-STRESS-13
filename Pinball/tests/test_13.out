857142864
142857144
